docs/guides/writing_stubs.rst

```diff
@@ -213,15 +213,28 @@ to use them freely to describe simple structural types.
 Incomplete Stubs
 ----------------
 
+When writing new stubs, it is not necessary to fully annotate all arguments,
+return types, and fields. Some items may be left unannotated or
+annotated with `_typeshed.Incomplete` (`documentation <https://github.com/python/typeshed/blob/main/stdlib/_typeshed/README.md>`_).::
```

Comment:
Could you fix the formatting? There's a stray period in front of the colon, and some terms (like `_typeshed.Incomplete` in this line) are surrounded with single backticks when it should be double backticks.

Reply:
Double ticks are needed to make things render as code. Jelle said in another comment that single ticks are the "default role", which I didn't know - I just checked the file preview.